src/libbson/src/bson/bson-decimal128.c

```diff
@@ -709,8 +709,8 @@ bson_decimal128_from_string_w_len (const char *string,     /* IN */
    }
 
    /* Encode significand */
-   significand_high = 0,   /* The high 17 digits of the significand */
-      significand_low = 0; /* The low 17 digits of the significand */
+   significand_high = 0;   /* The high 17 digits of the significand */
+   /* significand_low is the low 17 digits of the significand */
```

Comment:
Both `significant_high` and `significant_low` are already initialized to `0` on lines 503 and 504. Preserving the comments does not seem necessary either given similar the variable declarations are labeled with similar descriptions. Therefore, I believe lines 712-714 should be removed.

Reply:
Good point, done.